RecoHGCal/TICL/python/iterativeTICL_cff.py

```diff
@@ -6,7 +6,7 @@
 from RecoHGCal.TICL.HADStep_cff import *
 from RecoHGCal.TICL.ticlLayerTileProducer_cfi import ticlLayerTileProducer
 from RecoHGCal.TICL.ticlCandidateFromTrackstersProducer_cfi import ticlCandidateFromTrackstersProducer as _ticlCandidateFromTrackstersProducer
-from RecoHGCal.TICL.pfTICL_cfi import pfTICL
+from RecoHGCal.TICL.pfTICLProducer_cfi import pfTICLProducer as pfTICL
```

Comment:
This is something I am a bit suspicious about, but never checked explicitly.
If somebody else imports ```pfTICLProducer``` as is (```pfTICLProducer```) and modifies it, would this modification apply to what's called ```pfTICL``` here?
If that's the case, I'd rather we have an explicit clone here (like it's done for all other "import x as y".

Reply:
good point @slava77 
clone() added now

Reply:
Even though it was changed to a `clone()` already, let me clarify.

> This is something I am a bit suspicious about, but never checked explicitly.
> If somebody else imports `pfTICLProducer` as is (`pfTICLProducer`) and modifies it, would this modification apply to what's called `pfTICL` here?

Yes.

> If that's the case, I'd rather we have an explicit clone here (like it's done for all other "import x as y".

The core group recommends to modify configuration objects only in the cfi/cff files the objects are defined (a clone is counted as defining a new object, and this includes modifications with `cms.Modifier`s).